src/VisualStudio/Core/Def/Implementation/Venus/ContainedLanguage.cs

```diff
@@ -163,7 +163,10 @@ internal partial class ContainedLanguage<TPackage, TLanguageService>
 
             if (this.Project != null)
             {
-                documentId = this.Project.AddSourceTextContainer(SubjectBuffer.AsTextContainer(), filePath);
+                documentId = this.Project.AddSourceTextContainer(
```

Comment:
@jasonmalinowski can you confirmed that not adding document service is okay when this.project doesn't exist? should I create documentInfo with documentservice in those case?

Reply:
Ah, yes, I guess you'll need to add it in the other branch too, yes.

Reply:
As a note: my fix for LiveShare predates (from the perspective of the branch) the addition of document services, so there was nothing for me to do there.